scipy/stats/_continuous_distns.py

```diff
@@ -6337,6 +6337,23 @@ def func_all_free(floc, fshape):
             s_max = np.max(data) - np.min(data)
             u_min = np.min(data) - s_max
             u_max = (1-np.finfo(float).eps)*np.min(data)
+
+            # for some a > 1 cases, `u_max` needs to be decreased even more
+            # s.t. `func(u_min)` and `func(u_max)` have opposite signs.
```

Comment:
I still need to try to find values analytically for what the bracketing interval should be.